colin/core/config/config.py

```diff
@@ -125,6 +125,24 @@ def get_checks_path():
     return os.path.abspath(os.path.join(__file__, rel_path))
 
 
+def get_config_file(config):
+    """
+    Get the config file from name or path
+
+    :param config: str (name or path)
+    :return: str
+    """
+    if os.path.exists(config) and os.path.isfile(config):
+        return config
+
+    config_directory = get_config_directory()
```

Comment:
This should consume `config` variable, right?
And definition could be like `get_config_directory(config=None)`
and usage `get_config_directory(config=config)` for this case.

Reply:
No, it only finds the first existing config directory:
1. first: `$HOME/.local/share/colin/config`
2. second: `/usr/local/share/colin/config`

Reply:
Therefore description is misleading for option -c.
What about this: `Name of the configuration`
Path does not make sense here.

Reply:
For this state, it makes sense -- the `-c` accepts the name of the config or path to the config file.

But I will add another option for the file to make it clear.